distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1755,44 +1759,189 @@ public void testVideoThumbnail() throws Exception {
         }
     }
 
+    //TODO need to replace Library usage
```

Comment:
I plan to fix it in this PR, I just didn't have it sorted by the end of the day and wanted to get the review started.